New version: DaCosySheeep.FactDownloader version 1.1.0

DaCosySheeep opened the pull request:
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.5 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.5.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

---

 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/129152)

DaCosySheeep committed:
New version: DaCosySheeep.FactDownloader version 1.1.0

stephengillie commented:
Automatic Validation ended with:
>  Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Program Files\Fact downloader\Fact downloader.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)  ---> System.ComponentModel.Win32Exception (216): An error occurred trying to start process 'C:\Program Files\Fact downloader\Fact downloader.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.    --- End of inner exception stack trace ---    at System.Threading.Tasks.Task.ThrowIfExceptional(Boolean includeTaskCanceledExceptions) Exception during executable launch operation System.AggregateException: One or more errors occurred. (An error occurred trying to start process 'C:\Users\Validator\AppData\Local\Programs\Fact downloader\Fact downloader.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.)  ---> System.ComponentModel.Win32Exception (216): An error occurred trying to start process 'C:\Users\Validator\AppData\Local\Programs\Fact downloader\Fact downloader.exe' with working directory 'D:\TOOLS'. The specified executable is not a valid application for this OS platform.

DaCosySheeep committed:
Update DaCosySheeep.FactDownloader.installer.yaml

This release of Fact Downloader had trouble writing to files when installed for all users.

stephengillie commented:
@wingetbot waivers Add Validation-Executable-Error
